### PR TITLE
Fix session store regression #346

### PR DESCRIPTION
**Pull request by Gozala**
Fix for #346

### CODE REVIEW

File: src/browser/web-view.js [hunk 1]
```diff
@@ -255,9 +255,9 @@ define((require, exports, module) => {
       onTitleChange: event => edit(WebView.setTitle(event.detail)),
       onPrompt: event => console.log(event),
       onAuthentificate: event => console.log(event),
-      onScrollAreaChange: event =>
-        edit(WebView.setContentOverflows(event.detail.height >
-                                         event.target.parentNode.clientHeight)),
+      onScrollAreaChange: !state.contentOverflows && (event =>
```
Comment (gordonbrander) [inline src/browser/web-view.js]: Is this change related? Looks like a fix for scroll-away chrome.

Comment (Gozala) [inline src/browser/web-view.js]: > Is this change related? Looks like a fix for scroll-away chrome.

No it was not intended here, good catch! It's for the other bug #279

File: src/browser/actions.js [hunk 2]
```diff
@@ -95,14 +95,17 @@ define((require, exports, module) => {
   // Reads stored session. Returns either immutable data for the
   // session or null.
   const readSession = () => {
+    const session = localStorage[`session@${version}`]
```
Comment (gordonbrander) [inline src/browser/actions.js]: Nit: semicolon

**Comment by paulrouget** [comment]
Doesn't that restore the update banner?

**Comment by paulrouget** [comment]
See #352
